### PR TITLE
Add pipeline for tagged release

### PR DESCRIPTION
**Pull request by etienne-napoleone**
This PR aims to add pipeline for the tagged release.
- Publish a Github release with the tomo binary
- Push the Docker image for latest and the tag of the release

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -38,6 +40,22 @@ jobs:
     - while sleep 540; do echo "[ still running ]"; done &
     - go run build/ci.go test -coverage
     - kill %1
+
+  - stage: Github release
+    deploy:
+      provider: releases
+      api_key: $GITHUB_TOKEN
+      # name:
```
Comment (etienne-napoleone) [inline .travis.yml]: @ngtuna Any custom name?

File: .travis.yml [hunk 2]
```diff
@@ -38,6 +40,22 @@ jobs:
...
+      # body: (formating not preserved)
```
Comment (etienne-napoleone) [inline .travis.yml]: @ngtuna Should we put any content in the body?

File: .travis.yml [hunk 3]
```diff
@@ -38,6 +40,22 @@ jobs:
...
+      # prerelease: true
```
Comment (etienne-napoleone) [inline .travis.yml]: @ngtuna How can we make the difference between a release and pre-release?
Maybe we should use the draft option

File: .travis.yml [hunk 4]
```diff
@@ -38,6 +40,22 @@ jobs:
...
+      # draft: true
```
Comment (etienne-napoleone) [inline .travis.yml]: @ngtuna We could use that to add a manual validation step (add pre-release, check that everything's all right.

File: .travis.yml [hunk 5]
```diff
@@ -38,6 +40,22 @@ jobs:
     - while sleep 540; do echo "[ still running ]"; done &
     - go run build/ci.go test -coverage
     - kill %1
+
+  - stage: Github release
+    deploy:
+      provider: releases
+      api_key: $GITHUB_TOKEN
```
Comment (etienne-napoleone) [inline .travis.yml]: @thanhson1085 Can you add a github oauth token environment variable called `GITHUB_TOKEN` in the travis repo settings?

File: .travis.yml (hunk 1 above)
Comment (ngtuna) [reply .travis.yml]: Yes, keep `name` parameter there. We will update it on each release

File: .travis.yml (hunk 3 above)
Comment (ngtuna) [reply .travis.yml]: Eh... `prerelease` is correctly used. Mark it false when we make major release

File: .travis.yml (hunk 4 above)
Comment (ngtuna) [reply .travis.yml]: We can whenever update the release once it released

**Comment by etienne-napoleone** [comment]
Rebased and rename the released binary from tomo to tomo-linux-amd64

**Comment by etienne-napoleone** [comment]
@thanhson1085 Can you add a github oauth token environment variable called GITHUB_TOKEN in the travis repo settings?

**Comment by thanhson1085** [comment]
Yep, I added the token. lgtm.

**Comment by etienne-napoleone** [comment]
thank @thanhson1085 

@ngtuna I changed something on the deploy part which is not tested by Travis on PR. So feel free to merge it without waiting for the Travis check if you feel like it.

**Comment by ngtuna** [comment]
🎉

**Pull request merged**